testplan/common/utils/process.py

```diff
@@ -84,6 +87,57 @@ def kill_process(proc, timeout=5, signal_=None, output=None):
     return proc.returncode
 
 
+def kill_process_ex(proc, timeout=5, signal_=None, output=None):
```

Comment:
As discussed, please can we change the name of this function to something like `kill_process_psutil()` - to highlight the difference is that it takes a psutil object.